NorthstarDLL/mods/modmanager.cpp

```diff
@@ -264,6 +265,96 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 		}
 	}
 
+	int TotalCharLimit = 0;
+	// safe io
+	if (modJson.HasMember("SaveFiles"))
+	{
+		if (!modJson["SaveFiles"].IsArray())
+		{
+			spdlog::error("Save file list for {} is not an array!", Name);
+			return;
+		}
+		for (int i = 0; i < modJson["SaveFiles"].Size(); i++)
+		{
+			// MAX CHARACTERS PER MOD CHECK
+			// likely could be removed?
+			// hardcoded and not a startup arg because this limit is kinda ridiculous anyways.
+			// done to prevent a mod from pasting hundreds of file names to fill up the hard drive.
+			// this also means mods have to actually tell how many characters they're going to use instead of just typing in "9999999999".
+			// we check every time to prevent overflow from decieving us into thinking the character flow is low when it's actually just
+			// overflowed.
+			if (TotalCharLimit > 2000000)
```

Comment:
> The character limit and forced JSON file extension have been removed.  

isn't it still here?

Reply:
I forgot to push. Will do later today :P

Reply:
Done.